Reflections/Week_3_reflections.md

```diff
@@ -0,0 +1,29 @@
+## Accomplishments
+
+The only accomplishments this week that can be readily seen from the code that I have committed is that I have learned a number of ways to *not* write a beizer curve implementation. I had attempted to make an interface for beizer curves which matched the "generator" idiom of python. This was because I wanted a simple way to request N points over a beizer curve without having to keep track of the number of points that had been generated so far by the code using the interface.
```

Comment:
I disagree with this! I think your Bezier implementation is great, but rather the concern is placed on retrieving reliable paths from the bezier curves themselves. Using the generator idea was theoretically a fantastic idea, and something I can see myself doing as well, should I be faced with a similar predicament.